ingest/ingest_pipeline.py

```diff
@@ -487,7 +487,9 @@ def ingest_anndata(self):
         )
         if self.anndata.validate():
             self.report_validation("success")
-            if self.kwargs["extract_cluster"]:
+            if self.kwargs["extract_cluster"] == True:
```

Comment:
Previously, you had noticed that the job errors if `--obsm-keys` isn't set in the command line. I had meant to set up a happy path default so I've provided that here. If FOM requirements mean there will always be at least one expected clustering, we'd set that default here.